scipy/special/_cython_special.pxi

```diff
@@ -20,10 +20,10 @@ cdef public int wrap_PyUFunc_getfperr() nogil:
 cdef extern from "numpy/npy_math.h":
     double NPY_NAN
 
-cimport sf_error
-cimport _complexstuff
+from . cimport sf_error
+from . cimport _complexstuff
 cimport scipy.special._ufuncs_cxx
-from . import _ufuncs
+from scipy.special import _ufuncs
```

Comment:
Yeah, should be fine. Seems to pass now (modulo those errors that probably show up on untested systems, will write a mail to scipy-dev when we merge this, unless you are faster). And changing these two imports, seems no big deal to avoid the warning. Could probably clean up those np.errstate a bit, if you like